jmclient/jmclient/wallet.py

```diff
@@ -835,6 +840,34 @@ def set_next_index(self, mixdepth, internal, index, force=False):
         """
         raise NotImplementedError()
 
+    def clean_storage(self):
+        """Remove used scripts from boltzmann storage.
+        Utxo set must be synced"""
+
+        if self._storage.read_only:
+            return
+
+        scripts = []
+        utxo = self._utxos._utxo
```

Comment:
Never access a private property of another object. Private properties are not part of the object's public API.